src/Nest/CommonAbstractions/LazyDocument/LazyDocumentJsonConverter.cs

```diff
@@ -10,19 +10,24 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
 		{
 			var d = value as LazyDocument;
 			if (d?._Value == null)
+			{
+				writer.WriteNull();
```

Comment:
I think this is an oversight in the implementation; if `d` is `null` or `d.Value` is `null` and a value is not written to the writer for it, invalid json will be emitted